### PR TITLE
availableHorizontalScroll

### PR DESCRIPTION
**Pull request by hanorz**
I found a problem when i use your dragger and tableview delete function at the same time.
When i try to long press and moving tableviewcell, and HorizontalScroll it , the delete function will be active. So i want to block the HorizontalScroll when i am add delete function . 

Thanks for your libraries.

### CODE REVIEW

File: TableViewDragger/TableViewDragger.swift [hunk 1]
```diff
@@ -173,11 +175,22 @@ extension TableViewDragger {
...
+                    
+                    draggedCell.offset = point
+                    draggedCell.transformToPoint(point)
+                    draggedCell.location = gesture.location(in: tableView)
+                    
+                } else {
+                    
+                    draggedCell.offset = CGPoint(x: (actualCell?.frame.size.width)!/2, y: point.y)
+                    draggedCell.transformToPoint(CGPoint(x: (actualCell?.frame.size.width)!/2, y: point.y))
+                    draggedCell.location = CGPoint(x: (actualCell?.frame.size.width)!/2, y: gesture.location(in: tableView).y)
```
Comment (KyoheiG3) [inline TableViewDragger/TableViewDragger.swift]: Should use `draggedCell` or `if let` because forced unwrapping of `actualCell` is dangerous.

**Review by KyoheiG3 — COMMENTED** [review]
Thank you for your PR!!

File: TableViewDragger/TableViewDragger.swift (hunk 1 above)
Comment (hanorz) [reply TableViewDragger/TableViewDragger.swift]: Got it.

**Review by KyoheiG3 — APPROVED** [review]
Thanks!

**Pull request merged**